README.md

```diff
@@ -228,6 +228,21 @@ See [Ansible Using collections](https://docs.ansible.com/ansible/devel/user_guid
 
 See the [changelog](https://github.com/digitalocean/ansible-collection/tree/main/CHANGELOG.rst).
 
+## Releasing instructions
+
+Please review the [upstream documentation](https://docs.ansible.com/ansible/latest/community/collection_contributors/collection_releasing.html) for releasing Ansible collections.
+The utility for creating the release metadata is [antibull-changelog](https://github.com/ansible-community/antsibull-changelog) and can be install from [PyPI](https://pypi.org/project/antsibull-changelog/) with `pip install --user antsibull-changelog`.
+In general, the steps are as follow:
+
+1. Create a new branch for the release with `git checkout -b release/X.Y.Z`
+2. Bump the collection version in [galaxy.yml](./galaxy.yml) to `X.Y.Z`
+3. Run `antsibull-changelog release` to generate the release metadata
```

Comment:
```suggestion
3. Run `antsibull-changelog release` (or `poetry run antsibull-changelog release`) to generate the release metadata
```

Reply:
Could also make a `make release` that runs the poetry command for simplicity

Reply:
Yeah, let's see if we need to go down the `Makefile` path down the road.